### DISCUSSION TITLE
Unsure where I am going wrong, am I missing a step?

### DISCUSSION THREAD

**Discussion by cproo12** [firefly-iii/firefly-iii#5446] [Firefly III]
Hi there, I am running a stack via WAMPServer on Windows Server 2019

PHP 8.1.0 with every plugin listed enabled (I also had to enable Sodium for composer to let me install 5.6.8)

I can successfully download firefly-iii via composer with no issues, to the point where it says "Thank you for installing Firefly III, v5.6.8!"

Then I am not sure what to do. Obviously I need to update the .env file with my proper SQL server username/password. After doing that and running `php artisan migrate:refresh --seed` I get the error:

`Connection.php line 703: SQLSTATE[HY000] [1049] Unknown database 'firefly' (SQL: select * from information_schema.tables where table_schema
  = firefly and table_name = migrations and table_type = 'BASE TABLE')`

and

`Connection.php line 70: SQLSTATE[HY000] [1049] Unknown database 'firefly'`

So I try running `php artisan firefly-iii:create-databse` which says `Database "firefly" does not exist.
Created database "firefly"`

Then I run `php artisan migrate:refresh --seed` again and I get the following errors:

`Connection.php line 703: SQLSTATE[42S01]: Base table or view already exists: 1050 Table ''firefly'.'migrations'' already exists (SQL: create
   table 'migrations' ('id' int unsigned not null auto_increment primary key, migration varchar(191) not null, 'bat
  ch' int not null) default character set utf8mb4 collate 'utf8mb4_unicode_ci' engine = InnoDB)`

and

`Connection.php line 492: SQLSTATE[42S01]: Base table or view already exists: 1050 Table ''firefly'.'migrations'' already exists`

Happy to donate if you help me figure this out. I really want to try out this software but have been trying for the past week I've had off and can't make any progress :( tempted to go buy another server to use for linux just to try it as the demo looked so promising.

**Comment by JC5** [comment]
> Happy to donate if you help me figure this out

I appreciate the gesture but it's not necessary to bribe me into helping you. You should donate if you want to. And if you don't, also fine. My time is limited either way, donations are not my salary.

First of all: I don't understand why you have installed PHP 8.1.0 with all the packages, and the proceed to use Docker. The Docker container is called a "container" because it **contains** everything you need.

Have you tried the Docker Compose route?

https://docs.firefly-iii.org/firefly-iii/installation/docker/#using-docker-compose

**Reply by cproo12** [reply]
Sorry, I am just now seeing this. I was not trying to use docker or planning on using it, because I have never used it before. I am however familiar with Apache, Mysql, PHP, etc and already use them for my website on this server so I wanted to integrate everything in that way.

I guess my notifications on Github weren't on, so I figured I had no responses, gave up on windows, and now have it set up seemingly okay in an ubuntu 20.24 server... My only caveat is that I can't access the server from anything but localhost for the initial account registration, but the server is headless, and has no UI... Worst case scenario I will just have to install one and find a monitor/keyboard but currently just trying to find a way to complete initial setup via SSH/my windows PC.

EDIT: I have figured everything out! I could access it, but apache was using PHP 7.4 instead of the PHP 8.1 I installed manually. Changed that, ran into some permission problems, fixed those, and it is all working :)

Thank you so much for all the work and help you put into this software for the community around it! I truly will think about donating if I end up loving this :)
